edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/authenticators/DeviceScopeCertificateAuthenticator.cs

```diff
@@ -18,19 +14,15 @@ namespace Microsoft.Azure.Devices.Edge.Hub.CloudProxy.Authenticators
 
     public class DeviceScopeCertificateAuthenticator : DeviceScopeAuthenticator<ICertificateCredentials>
     {
-        readonly string iothubHostName;
         readonly IList<X509Certificate2> trustBundle;
 
         public DeviceScopeCertificateAuthenticator(
             IDeviceScopeIdentitiesCache deviceScopeIdentitiesCache,
-            string iothubHostName,
```

Comment:
This was added so that we could authenticate module certs. I know we removed that auth capability but it is imminent that we will add this back as we integrate with IoT Hub.

Reply:
Sure, in that case we can add it back.. should not be too hard.